api/src/main/java/org/openmrs/Person.java

```diff
@@ -65,7 +65,7 @@
 @Entity
 @Table(name = "person")
 @Inheritance(strategy = InheritanceType.JOINED)
-public class Person extends BaseChangeableOpenmrsData {
+public class Person extends BaseCustomizableData<PersonAttribute> implements Customizable<PersonAttribute> {
```

Comment:
Why do you implementing `Customizable<PersonAttribute>` when `BaseCustomizableData<A>` already implements that interface? 🤔

Reply:
hei @dkayiwa   according to some comments here and the first comment i made,do we still need this ticket??